tests/smoke/install/task.yaml

```diff
@@ -9,6 +9,8 @@ debug: |
 restore: |
     rm -f stderr.log
     rm -f stdout.log
+    # requied! in autopkgtest no suite restore is run at all
```

Comment:
Thanks for this! I always claimed that we should be careful and write real restore sections rather than having to rely on broad restore that some suites implement simply because it's less magical and more robust this way. Now we learned it is also required for test mobility in and out of some suites.